src/main/archetype/it.tests/pom.xml

```diff
@@ -17,138 +17,194 @@
+            <groupId>com.adobe.cq</groupId>
+            <artifactId>aem-cloud-testing-clients</artifactId>
+            <version>0.2.3</version>
         </dependency>
+#elseif ($aemVersion.startsWith("6.5"))
         <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-api</artifactId>
-            <scope>provided</scope>
+            <groupId>com.adobe.cq</groupId>
+            <artifactId>cq-testing-clients-65</artifactId>
+            <version>1.1.1</version>
         </dependency>
+#elseif ($aemVersion.startsWith("6.4"))
         <dependency>
-            <groupId>com.adobe.aem</groupId>
-            <artifactId>uber-jar</artifactId>
-            <classifier>apis</classifier>
-            <scope>provided</scope>
+            <groupId>com.adobe.cq</groupId>
+            <artifactId>cq-testing-clients-64</artifactId>
+            <version>0.1.1</version>
         </dependency>
 #else
         <dependency>
-            <groupId>com.adobe.aem</groupId>
-            <artifactId>aem-sdk-api</artifactId>
+            <groupId>com.adobe.cq</groupId>
+            <artifactId>cq-testing-clients-63</artifactId>
+            <version>0.1.0</version>
         </dependency>
 #end
         <dependency>
-            <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter</artifactId>
-            <scope>provided</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.junit.vintage</groupId>
-            <artifactId>junit-vintage-engine</artifactId>
-            <scope>provided</scope>
-        </dependency>
-        <dependency>
-            <!-- TODO create separate API package, this is just to get the special Sling test runner -->
-            <groupId>org.apache.sling</groupId>
-            <artifactId>org.apache.sling.junit.core</artifactId>
-            <version>1.0.8</version>
-            <scope>provided</scope>
-        </dependency>
-
-        <dependency>
-            <groupId>${groupId}</groupId>
-            <artifactId>${rootArtifactId}.core</artifactId>
-            <version>${project.version}</version>
-            <scope>provided</scope>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+            <version>1.7.25</version>
         </dependency>
     </dependencies>
+
+    <profiles>
+        <profile>
+            <id>local</id>
+            <properties>
+                <!-- Cloud author service (LB) -->
+                <cloud.author.url>http://localhost:4502</cloud.author.url>
+                <cloud.author.user>admin</cloud.author.user>
+                <cloud.author.password>admin</cloud.author.password>
+
+                <!-- cloud publish service (LB) -->
+                <cloud.publish.url>http://localhost:4503</cloud.publish.url>
+                <cloud.publish.user>admin</cloud.publish.user>
+                <cloud.publish.password>admin</cloud.publish.password>
```

Comment:
wondering if the names of these properties aren't too confusing (local profile - cloud properties?!). What about we go with `it.author.url` and so forth (`s/cloud/it`)?

Reply:
Fixed at 84bdef7.